lib/moonshot/controller.rb

```diff
@@ -41,7 +118,8 @@ def status
     end
 
     def deploy_code
-      version = "#{stack_name}-#{Time.now.to_i}"
+      version = [@config.app_name, @config.environment_name, Time.now.to_i]
+                .join('-')
```

Comment:
Perhaps, out of scope of this story. This just keeps compatibility with the current naming.